src/Discord/MessageSender.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+/*
+ * This file is part of the Slack Secret Santa project.
+ *
+ * (c) JoliCode <coucou@jolicode.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Joli\SlackSecretSanta\Discord;
+
+use Joli\SlackSecretSanta\SecretSanta;
+
+class MessageSender
+{
+    /** @var DiscordService */
+    private $discordService;
+
+    public function __construct(DiscordService $discordService)
+    {
+        $this->discordService = $discordService;
+    }
+
+    public function sendSecretMessage(SecretSanta $secretSanta, string $giver, string $receiver): void
+    {
+        $text = sprintf(
+'Hi! You have been chosen to be part of a Secret Santa!
```

Comment:
We have to find a way to share this text between all the apps, to avoid duplication.

Reply:
Yep I thought about it but format is not always the same. For example, bold and link have been tweaked for Discord

Reply:
Using translation ? So you can, in future, even throwing some localized message ?